apps/design-system/src/pages/view-preview/view-preview.tsx

```diff
@@ -20,6 +22,16 @@ const views: Record<string, ReactNode> = {
     <RootViewWrapper>
       <RepoListWrapper />
     </RootViewWrapper>
+  ),
+  'pull-request-list': (
+    <RootViewWrapper>
+      <PullRequestListWrapper />
+    </RootViewWrapper>
+  ),
+  'pull-request-compare': (
+    <RootViewWrapper>
+      <PullRequestCompareWrapper />
+    </RootViewWrapper>
```

Comment:
It might be better to display these using the `RepoViewWrapper` instead of the `RootWrapper` to get some more of the Repo goodness.
```suggestion
    <RepoViewWrapper>
      <PullRequestListWrapper />
    </RepoViewWrapper>
  ),
  'pull-request-compare': (
    <RepoViewWrapper>
      <PullRequestCompareWrapper />
    </RepoViewWrapper>
```